src/nvim/macros.h

```diff
@@ -226,4 +226,10 @@
 
 #define EMPTY_POS(a) ((a).lnum == 0 && (a).col == 0 && (a).coladd == 0)
 
+// Mark unused variables with _UNUSED, so that we can suppress warnings from
+// gcc's -Wunused-value and -Wunused-but-set-variable.
+// Prefix with _ to not clash with xdiff/xinclude.h which defines UNUSED for use
```

Comment:
mpack is a separate package, which we just happen to vendor currently.  I don't think we need to add this to nvim's headers.